extensions/civetweb/processors/ListenHTTP.h

```diff
@@ -155,18 +160,15 @@ class ListenHTTP : public core::Processor {
   // Write callback for transferring data from HTTP request to content repo
   class WriteCallback : public OutputStreamCallback {
    public:
-    WriteCallback(struct mg_connection *conn, const struct mg_request_info *reqInfo);
+    WriteCallback(std::unique_ptr<io::BufferStream>);
     int64_t process(std::shared_ptr<io::BaseStream> stream);
 
    private:
-    // Logger
     std::shared_ptr<logging::Logger> logger_;
-
-    struct mg_connection *conn_;
-    const struct mg_request_info *req_info_;
+    std::shared_ptr<io::BufferStream> request_content_;
```

Comment:
could this be a `unique_ptr` instead of a `shared_ptr`?

Reply:
Replaced in [385be9e](https://github.com/apache/nifi-minifi-cpp/pull/921/commits/385be9e942f776f41fe87ab2526fbe016b8d1d51)